pypesto/history/csv.py

```diff
@@ -87,16 +88,16 @@ def _update_counts_from_trace(self) -> None:
     def update(
         self,
         x: np.ndarray,
-        sensi_orders: Tuple[int, ...],
+        sensi_orders: tuple[int, ...],
         mode: ModeType,
         result: ResultDict,
     ) -> None:
-        """See `History` docstring."""
+        """See :class:`History` docstring."""
```

Comment:
```suggestion
        """See :meth:`History.update`."""
```
A direct link will be more helpful.

Same for all methods below.